media/css/products/vpn/resource-center.scss

```diff
@@ -0,0 +1,167 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/inline-list';
+@import '~@mozilla-protocol/core/protocol/css/components/article';
+@import '~@mozilla-protocol/core/protocol/css/components/call-out';
+@import '~@mozilla-protocol/core/protocol/css/components/card';
+@import '~@mozilla-protocol/core/protocol/css/components/split';
+@import '~@mozilla-protocol/core/protocol/css/templates/card-layout';
+@import '~@mozilla-protocol/core/protocol/css/includes/fonts/metropolis';
+@import '~@mozilla-protocol/core/protocol/css/includes/fonts/inter';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-vpn';
+@import '~@mozilla-protocol/core/protocol/css/components/breadcrumb';
+
+.resource-center-page-header {
+    .mzp-l-content {
+        max-width: $content-md;
+        .mzp-c-call-out-title {
+            @include font-firefox;
+        }
+        .mzp-c-call-out-desc {
+            font-weight: 700;
+            margin-left: auto;
+            margin-right: auto;
+            max-width: $content-sm;
+        }
+    }
+    @media #{$mq-md} {
+        background-image: url('/media/img/products/vpn/resource-center/moz_resource_center_header_bg.svg');
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: 90% 100%;
+    }
+    @media #{$mq-lg} {
+        background-position: 45% $layout-xs;
+        background-size: auto;
+    }
+}
+
+.resource-center-hero {
+    background-color: $color-white;
+}
+
+.resource-center-wordmark {
+    @include image-replaced();
+    @include background-size(94px 32px);
+    background: transparent url('/media/img/logos/vpn/logo-with-wordmark.svg') top left no-repeat;
+    height: 32px;
+    margin: 0 auto $spacing-lg;
+    width: 94px;
+
+    @media #{$mq-md} {
+        margin: 0 0 $spacing-xl;
+    }
+
+}
+
+.resource-center-cta {
+    margin-bottom: $spacing-md;
+}
+
+.resource-center-split-media .mzp-c-split-media-asset {
+    // smaller image on small breakpoints
+    max-width: 80%;
+
+    @media #{$mq-md} {
+        max-width: 100%;
+    }
+}
+
+.resource-center-cta-title {
+    margin-bottom: $spacing-2xl;
+}
+
+.resource-center-cta-desc {
+    @include text-body-sm;
+    @include font-base;
+    color: $color-marketing-gray-60;
```

Comment:
Looks like this color contrast on this link does not quite meet a11y guidelines:

<img width="357" alt="image" src="https://user-images.githubusercontent.com/400117/141772562-2b53a2d1-b921-4acd-b5bb-89160aa62c2c.png">

Reply:
oh, good catch!